src/App.css

```diff
@@ -55,23 +55,26 @@
 }
 /************side-menu*************/
 .aside-list-icons{
- display: flex;
- flex-direction: column;
- justify-content: center;
- align-items: center;
- gap: 3rem;
- list-style: none;
- font-size: 3rem;
- color: var(--PRIMERY-color);
+display: flex;
+flex-direction: column;
+justify-content: center;
+align-items: center;
+gap: 3rem;
+margin-left: 7rem; 
+list-style: none;
+font-size: 3rem;
+color: var(--PRIMERY-color);
 }
+
 .icons-bar{
   font-size: 5rem;
+  color: var(--PRIMERY-color);
 }
 .main-hero-section{
     margin-top: .2rem;
```

Comment:
For less than 1 rem use px because you don't need to scale any element whose size < 1rem